engine-nextgen/src/main/kotlin/graphql/nadel/validation/NadelSchemaValidationError.kt

```diff
@@ -317,6 +317,25 @@ sealed interface NadelSchemaValidationError {
         override val subject = overallField
     }
 
+    data class NonExistentHydrationActorFieldArgument(
+            val parentType: NadelServiceSchemaElement,
+            val overallField: GraphQLFieldDefinition,
+            val hydration: UnderlyingServiceHydration,
+            val actorServiceQueryType: GraphQLObjectType,
+            val argument: String,
+    ) : NadelSchemaValidationError {
+        val service: Service get() = parentType.service
+
+        override val message = run {
+            val of = makeFieldCoordinates(parentType.overall.name, overallField.name)
+            val s = hydration.serviceName
+            val af = "${actorServiceQueryType.name}.${hydration.pathToActorField.joinToString(separator = ".")}"
+            "Hydration on field $of is missing argument $argument on hydration actor $s.$af"
```

Comment:
This is the other way around? The class names should be flipped.

Also:

```suggestion
            "Hydration on field $of is missing the required argument $argument on hydration actor $s.$af"
```